.circleci/config.yml

```diff
@@ -51,14 +51,10 @@ commands:
 jobs:
   build-linux:
     docker:
-      - image: cimg/openjdk:11.0.11-browsers@sha256:02d8ac1beb99bff9c303376388355c6f87f9e29960804e920f01f30547214f7c
-    resource_class: medium+
+      - image: cimg/openjdk:17.0.7-browsers@sha256:52a950f68f31d7e947e9468e11acb38177272a59c446044d13f0c6d7d66d12ed
+    resource_class: large
```

Comment:
The daemon kept dying when running with medium+

I'm not sure if that would get fixed now since the cache is populated.